packages/block-editor/src/components/iframe/index.js

```diff
@@ -261,9 +261,6 @@ function Iframe( {
 		${ styles }
 		${ scripts }
 	</head>
-	<body>
-		<script>document.currentScript.parentElement.remove()</script>
-	</body>
```

Comment:
@tyxla, what's the reason for removing this logic?

I think this is the cause of at least one e2e test failure. Ref #52405.

Reply:
Back when I was testing with the beta, this was an essential fix to make the editor work (see https://github.com/WordPress/gutenberg/pull/61521/commits/7936e41e4058752881717b4a25826d3216a14cf2 for the full fix). Without it, the editor was crashing during its initial boot.

Reply:
I appreciate your work on it, @Mamaduka! Feel free to revert it and seek a different fix. I've been delayed by other things, but hoping to get back to this one soon.